pandasai/llm/cohere.py

```diff
@@ -0,0 +1,111 @@
+"""Cohere LLM"""
+
+import os
+from dotenv import load_dotenv
+import cohere
+from .base import LLM
+from ..exceptions import APIKeyNotFoundError, UnsupportedCohereModelError
+
+load_dotenv()
+
+
+# pylint: disable=duplicate-code
+class CohereLLM(LLM):
+    """Cohere LLM"""
+
+    _supported_chat_models = [
+        "command",
+        "command-nightly",
+        "command-light",
+        "command-light-nightly",
+        "command-xlarge",
+        "command-xlarge-nightly",
+    ]
+
+    api_token: str
+    model: str = "command-xlarge-nightly"
+    temperature: float = 0
+    max_tokens: int = 512
+    top_p: float = 1
+    frequency_penalty: float = 0
+    presence_penalty: float = 0.6
+    stop: str = None
+
+    def __init__(
+        self,
+        api_token: str = None,
+        **kwargs,
+    ):
+        self.api_token = api_token or os.getenv("COHERE_API_KEY")
+        if self.api_token is None:
```

Comment:
Could you also add this variable to the `.env.example` file?

Reply:
Done!